src/app/logproc/hashfarm_wordlists.ml

```diff
@@ -0,0 +1,321 @@
+(** Long lists of words. *)
+
+let adjectives =
```

Comment:
Can we have an assert that checking that each category contains no duplicate words?

Reply:
Are these enough words? It seems like the number of combination of words is over a million, around 1109400. Would it be better to have one more column?

Reply:
We can expect to have roughly `sqrt(length combined_wordlist)` names before a collision. Right now, that's about 1k names. If we made the list 100 bigger (added a fourth column with 100 entries), that number goes up to 10k, and the table would be 2GB. I'll use a more efficient algorithm for generating/indexing the wordlist. Any suggestions for the fourth wordlist? :)

Reply:
How about having the 4th column be verbs? So, we can have something like: `nifty beige yak fishes`

Reply:
@johnwu93  @cmr can we implement with 1:1M chance for now?  I don't see this being used that much, but it has value as it is.